### DISCUSSION TITLE
Does Avalonia have a discord server?

### DISCUSSION THREAD

**Discussion by MordechaiHadad** [AvaloniaUI/Avalonia#11892] [Q&A] [answered]
If it does I can't really find it

**Comment by hez2010** [comment]
Avalonia may not have a discord server, but it has an official telegram group.

**Comment by timunie** [comment, marked as answer]
[Telegram](https://t.me/Avalonia) is preffered, gitter also works somewhat. Other than that you can also use Discussions for "larger" topics.

**Reply by MordechaiHadad** [reply]
I see but why not open a discord server? most people have a discord account(i think based on anecdotal evidence)

**Reply by timunie** [reply]
we have one, but just no-one is there. We have not enough capacity to mange all of them. We used to have a bridge from telegram to gitter (still working) and to discord, but the later one we are not able to get it back running unfortunately. Everyone has a different opinion what chat suits them most, and we made the decision to use telegram as our main chat. I hope for your understanding. 

See also: https://github.com/AvaloniaUI/Avalonia/discussions/11369 for our users votes.

**Reply by MordechaiHadad** [reply]
I see thank you!
